e2e-environment/README.md

```diff
@@ -41,13 +45,13 @@ docker run --detach --rm \
   --publish 127.0.0.1:5001:5001 \
   --publish 127.0.0.1:5002:5002 \
   --publish 127.0.0.1:8545:8545 \
-  raidennetwork/lightclient-e2e-environment \
+  raidennetwork/lightclient-e2e-environment
 ```
 
 ## Access Services
 
-After starting the Docker container as described above, the services are
-accessible at the following ports on `localhost`:
+After the the Docker container go started, the services are accessible at the
```

Comment:
`s/the the/the/`
`s/go started/get started/`

Reply:
Haha, you know that GitHub supports review suggestions? But copy pasting the commands to Vim is also nice. :rofl:

Reply:
:heavy_check_mark: